cypress/integration/firstTests.spec.js

```diff
@@ -1,7 +1,49 @@
 /// <reference types="cypress" />
 
-describe('', () => {
-  it('', () => {
+const { generateUser } = require('../support/generate');
+
+describe('Sign in', () => {
+  beforeEach(() => {
+    cy.visit('https://react-redux.realworld.io/#/login')
```

Comment:
Don't forget to put ; to the end of line
```suggestion
    cy.visit('https://react-redux.realworld.io/#/login');
```